airflow/providers/airbyte/hooks/airbyte.py

```diff
@@ -46,12 +46,12 @@ class AirbyteHook(HttpHook):
     ERROR = "error"
     INCOMPLETE = "incomplete"
 
-    def __init__(self, airbyte_conn_id: str = "airbyte_default", api_version: Optional[str] = "v1") -> None:
+    def __init__(self, airbyte_conn_id: str = "airbyte_default", api_version: str = "v1") -> None:
         super().__init__(http_conn_id=airbyte_conn_id)
         self.api_version: str = api_version
 
     def wait_for_job(
-        self, job_id: str, wait_seconds: Optional[float] = 3, timeout: Optional[float] = 3600
+        self, job_id: str, wait_seconds: float = 3, timeout: Optional[float] = 3600
```

Comment:
We should probably do this too

```suggestion
        self, job_id: Union[str, int], wait_seconds: float = 3, timeout: Optional[float] = 3600
```

Reply:
changed, thanks